app/controllers/price_policies_controller.rb

```diff
@@ -33,28 +31,36 @@ def index
 
   # GET /facilities/:facility_id/{product_type}/:product_id/price_policies/new
   def new
-    @start_date = new_start_date
-    @expire_date = set_max_expire_date
+    # If there are active policies, start tomorrow. If none, start today
+    @start_date = Date.today + (active_policies? ? 1 : 0)
```

Comment:
I think `active_policies? ? Date.tomorrow : Date.today` may be clearer here, and not even need the explanatory comment above.

Reply:
Good call. That is definitely clearer. This was just an inlining of a separate method call, and I think that's probably even better than the method call.